src/template-errors.js

```diff
@@ -45,6 +45,14 @@ class AbstractForbiddenError extends AbstractError {
   }
 }
 
+class AbstractUnauthorizedError extends AbstractError {
+  get statusCode() { return Restypie.Codes.Unauthorized; }
```

Comment:
Confusion : `Unauthorized` actually means "Requires authentication". If the caller doesn't have the authorization to perform an operation, then the status code to send back is `Forbidden`. (I know...)